client/my-sites/importer/importer-blogger.jsx

```diff
@@ -0,0 +1,47 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FileImporter from './file-importer';
+
+/**
+ * Module variables
+ */
+const importerData = {
+	title: 'Blogger.com',
+	icon: 'blogger',
+};
+
+class ImporterBlogger extends React.PureComponent {
+	static displayName = 'ImporterBlogger';
+
+	render() {
+		importerData.description = this.props.translate(
+			'Import posts, pages, comments, tags and images ' + 'from a Blogger.com export file.'
+		);
+
+		importerData.uploadDescription = this.props.translate(
+			'Upload a {{b}}Blogger.com export file{{/b}} to start ' +
+				'importing into {{b2}}%(title)s{{/b2}}.',
+			{
+				args: { title: this.props.site.title },
```

Comment:
We should add `site` as a required prop using `propTypes` (making use of `shape` to ensure it has a `title` too)

Reply:
Thanks - yep, agree with the comments. I'll make sure to replicate the improvements to importer-medium.jsx, which I took a lot of the code from.

I did find missing propTypes a bit confusing, so will make sure to add them!

Reply:
Thanks @talldan :)

My rule of thumb is if it's chained (and thus could through a missing ref error) it's worth either being mentioned in propTypes or using `get` instead to safely grab the chained value :)